app/assets/javascripts/controllers/middleware_providers/_module.js

```diff
@@ -0,0 +1,50 @@
+miqHttpInject(angular.module('middleware.provider', ['miQStaticAssets', 'ui.bootstrap', 'ui.router', 'patternfly.select', 'ui.bootstrap.tabs', 'patternfly.views', 'ngAnimate']))
+.config(function($stateProvider, $locationProvider, $urlRouterProvider) {
+  $stateProvider.state('list_providers', {
+    url: '/ems_middleware/show_list',
+    templateUrl: '/static/middleware/list_providers.html',
+    controller: 'miqListProvidersController as vm'
```

Comment:
ok thanks. in miq it's even more misleading as vm also stands for virtual machine :)